locale.typ

```diff
@@ -0,0 +1,116 @@
+#let TITLEPAGE_SECTION_A = (
+  "de": "für den Erwerb des",
+  "en": "for the",
+)
+
+#let TITLEPAGE_SECTION_B = (
+  "de": "aus dem Studiengang ",
+  "en": "from the course of studies ",
+)
+
+#let TITLEPAGE_SECTION_C = (
+  "de": "an der ",
+  "en": "at the ",
+)
+
+#let TITLEPAGE_STUDENT_ID = (
+  "de": "Matrikelnummer, Kurs:",
+  "en": "Student ID, Course:",
+)
+
+#let TITLEPAGE_COMPANY = (
+  "de": "Unternehmen:",
+  "en": "Company:",
+)
+
+#let TITLEPAGE_COMPANY_SUPERVISOR = (
+  "de": "Betreuer im Unternehmen:",
+  "en": "Supervisor in the Company:",
+)
+
+#let TITLEPAGE_SUPERVISOR = (
+  "de": "Betreuer an der ",
+  "en": "Supervisor at ",
+)
+
+#let DECLARATION_OF_OWNERSHIP_TITLE = (
+  "de": "Selbstständigkeitserklärung",
+  "en": "Declaration of Authorship",
+)
+
+#let DECLARATION_OF_OWNERSHIP_SECTION_A_SINGLE = (
+  "de": "Gemäß Ziffer 1.1.13 der Anlage 1 zu §§ 3, 4 und 5 der Studien- und Prüfungsordnung für die Bachelorstudiengänge im Studienbereich Technik der Dualen Hochschule Baden- Württemberg vom 29.09.2017. Ich versichere hiermit, dass ich meine Arbeit mit dem Thema:",
+  "en": "In accordance with section 1.1.13 of Annex 1 to §§ 3, 4 and 5 of the study and examination regulations for the Bachelor's degree programs in the field of technology at the Baden-Wuerttemberg Cooperative State University dated 29.09.2017. I hereby affirm that I have written my thesis on the topic:"
+)
+
+#let DECLARATION_OF_OWNERSHIP_SECTION_A_PLURAL = (
+  "de": "Gemäß Ziffer 1.1.13 der Anlage 1 zu §§ 3, 4 und 5 der Studien- und Prüfungsordnung für die Bachelorstudiengänge im Studienbereich Technik der Dualen Hochschule Baden- Württemberg vom 29.09.2017. Wir versichern hiermit, dass wir unsere Arbeit mit dem Thema:",
+  "en": "In accordance with section 1.1.13 of Annex 1 to §§ 3, 4 and 5 of the study and examination regulations for the Bachelor's degree programs in the field of technology at the Baden-Wuerttemberg Cooperative State University dated 29.09.2017. We hereby affirm that we have written our thesis on the topic:",
+)
+
+#let DECLARATION_OF_OWNERSHIP_SECTION_B_SINGLE = (
+  "de": "selbstständig verfasst und keine anderen als die angegebenen Quellen und Hilfsmittel benutzt habe. Ich versichere zudem, dass die eingereichte elektronische Fassung mit der gedruckten Fassung übereinstimmt.",
+  "en": "independently and have not used any sources or aids other than those specified. I also confirm that the electronic version submitted is identical to the printed version."
+)
+
+#let DECLARATION_OF_OWNERSHIP_SECTION_B_PLURAL = (
+  "de": "selbstständig verfasst und keine anderen als die angegebenen Quellen und Hilfsmittel benutzt haben. Wir versichern zudem, dass die eingereichte elektronische Fassung mit der gedruckten Fassung übereinstimmt.
+",
+  "en": "independently and have not used any sources or aids other than those specified. We also confirm that the electronic version submitted is identical to the printed version.",
+)
```

Comment:
The declaration of authorship is required to be in German according to the DHBW guidelines, so the English version is not needed here.

Reply:
In this case, should I hard-code it to use the German version?
Currently, it uses whatever language the user sets in the configuration.

Reply:
See how I changed it here https://github.com/typst/packages/pull/865.

But I think it makes sense to have a way of overwriting the default as you say.

Reply:
I already have a parameter in the template to provide a custom confidentiality statement. I think it makes sense to add a parameter to overwrite the whole declaration of authorship to provide your own custom one in the same way.